exercises/gigasecond/description.md

```diff
@@ -1,3 +1,4 @@
-Calculate the moment when someone has lived for 10^9 seconds.
+Given a moment, determine the moment that would be after a gigasecond
```

Comment:
I would have said `as input`, but if you look at all the other instances of "given" in this repository, "as input" is very rare. I suppose the word "given" is given to imply "as input"

Reply:
I think "given" -> "given as input" is clear.